clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/config/BootstrapKubernetesConfigurationProvider.java

```diff
@@ -90,17 +89,18 @@ public KubernetesConfigurationProperties getKubernetesConfigurationProperties(
           .getAccounts()
           .add((KubernetesConfigurationProperties.ManagedAccount) result.get());
     }
-
-    try {
-      propertiesMap.remove("accounts"); // accounts are already processed above
-      result = bind(getFlatMap(propertiesMap), KubernetesConfigurationProperties.class);
-      k8sConfigProps.setRawResourcesEndpointConfig(
-          ((KubernetesConfigurationProperties) result.get()).getRawResourcesEndpointConfig());
-      // mappings to any future fields similar to rawResourcesEndpointConfig go here
-    } catch (NoSuchElementException e) {
-      // this binding error occurs when rawResourcesEndpointConfig is not supplied.
-      // ignoring it as rawResourcesEndpointConfig is not a mandatory configuration
-    }
+    /*
+        try {
+          propertiesMap.remove("accounts"); // accounts are already processed above
+          result = bind(getFlatMap(propertiesMap), KubernetesConfigurationProperties.class);
+          k8sConfigProps.setRawResourcesEndpointConfig(
+              ((KubernetesConfigurationProperties) result.get()).getRawResourcesEndpointConfig());
+          // mappings to any future fields similar to rawResourcesEndpointConfig go here
+        } catch (NoSuchElementException e) {
+          // this binding error occurs when rawResourcesEndpointConfig is not supplied.
+          // ignoring it as rawResourcesEndpointConfig is not a mandatory configuration
+        }
+    */
```

Comment:
Seems that this block of code can just be deleted since `rawResourcesEndpointConfig` should already be included per account.